docs/daily/2024_12_16.md

```
@@ -0,0 +1,36 @@
+- git commit
+    - 현재까지의 변경 사항을 기록
+    - -m "커밋 메세지"
+- git clone
+    - 원격 저장소의 내용을 복제하여 새 프로젝트 생성
+- git pull
+    - 원격 저장소의 해당 브랜치의 맨 마지막 커밋의 코드를 로컬에 불러옴
+- git push
+    - 지금까지 작성 된 commit들을 원격 저장소에 기록
+
+## 브랜치란?
+![](https://lh7-rt.googleusercontent.com/slidesz/AGV_vUcBXXX3P1ATDti-KdAWJch6zZcCcGlxVSA4rplgJ8cHENskrsW9Iz0XofY04XarzLedn-DefF9ndv70wLQgy_NNBTk_Ckny1gMr6hi29DD6p5eXwbKkNdeaSJABr9PAhZ6Fzv64tbvV1KfIhUqEdCsX=s2048?key=CnSYR3bHv9n7fmrGx7oyHQ)
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)